package.json

```diff
@@ -1,7 +1,7 @@
 {
   "private": true,
   "scripts": {
-    "lint": "markdownlint src/**/*.md",
+    "lint": "markdownlint \"src/**/*.md\"",
```

Comment:
Without the quotes, this is just a Bash glob that should expand to the same list of files. That is,

```sh
echo src/**/*.md
```

…in the root of the repo prints the list of files, including e.g. `src/404.md`. Therefore, I wouldn’t have expected this change to make a difference.

Why does it?

Reply:
I think without the quotes it is treated as a glob that only treats the "**" as a single folder level glob, with it it does expand it. Not saying it hasn't bit me a few times before, but I've seen this behaviour on other NodeJS CLI tools